openff/interchange/components/potentials.py

```diff
@@ -2,9 +2,10 @@
 
 import ast
 import json
-from typing import Annotated, Any
+from typing import TYPE_CHECKING, Annotated, Any
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Annotated, Any, TypeAlias
```

Needed for some suggestions below.